apps/hash-ai-worker-ts/src/activities/flow-activities/shared/infer-facts-from-text/infer-entity-facts-from-text.ts

```diff
@@ -110,6 +110,11 @@ const systemPrompt = dedent(`
 
   These facts will be later used to construct the entities with the properties and links which the user will specify.
   If any information in the text is relevant for constructing the relevant properties or outgoing links, you must include them as facts.
+  
+  Each fact should be in the format <subject> <predicate> <object>, where the subject is the singular subject of the fact.
+  Example:
+  [{ text: "Company X acquired Company Y.", prepositionalPhrases: ["in 2019", "for $10 million"], subjectEntityLocalId: "abc", objectEntityLocalId: "123" }]
+  Don't include facts which start with a subject you can't provide an id for.
```

Comment:
I'm sure you're aware, but would be interesting to see if this is any more impactful when inserted as a synthetic example (user and assistant message), where the user message includes some text including this fact about companies X and Y.

Reply:
Yes – I want to experiment more with examples here, as it's one of the places I expect we will need to focus on improvements most.